src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -891,6 +891,25 @@ public function testAutowiring()
 
         $this->assertEquals('a', (string) $container->getDefinition('b')->getArgument(0));
     }
+
+    /**
+     * @group legacy
```

Comment:
As this test and the other one should not be removed in 4.0, I suggest to add a comment to make it clear.